back-end/resources/room_config.json

```diff
@@ -0,0 +1,251 @@
+{
+  "general": {
+    "name": "Escape X",
+    "duration": "00:30:00",
+    "host": "192.0.0.84",
+    "port": 1883
+  },
+  "devices": [
+    {
+      "id": "controlBoard",
+      "description": "Control board with three switches, three slides with lights and one main switch.",
+      "input": {
+        "redSwitch": {
+          "type": "boolean"
+        },
+        "orangeSwitch": {
+          "type": "boolean"
+        },
+        "greenSwitch": {
+          "type": "boolean"
+        },
+        "slider1": {
+          "type": "integer"
+        },
+        "slider2": {
+          "type": "integer"
+        },
+        "slider3": {
+          "type": "integer"
+        },
+        "mainSwitch": {
+          "type": "boolean"
+        }
+      },
+      "output": {
```

Comment:
Example suggestion:

"devices": [
{
  "id": "controlBoard",
  "description": "Control board with three switches, three slides with lights and one main switch.",
  "input": {
	"redSwitch": "boolean",
	"orangeSwitch": "boolean",
	"greenSwitch": "boolean",
	"slider1": "integer",
	"slider2": "integer",
	"slider3": "integer",
  },
  "output": {
	"greenLight1": "string",
	"greenLight2": "string",
	"greenLight3": "string",
	"redLight1": "string",
	"redLight2": "string",
	"redLight3": "string",
  },
  "instructions: {
	"interval": "integer",
    "delay": "integer",
  }
}

Reply:
type = instruction
interval = 6
id/component = greenLight1

Reply:
type = instruction
greenLight1 =  off